src/modules/augmintTokenProvider.js

```diff
@@ -30,11 +31,29 @@ export default () => {
     return;
 };
 
-const setupListeners = () => {
-    const augmintToken = store.getState().contracts.latest.augmintToken.ethersInstance;
+const setupContractEventListeners = async () => {
+    processedContractEvents = {};
 
-    augmintToken.on("AugmintTransfer", (...args) => {
-        onAugmintTransfer(...args);
+    const augmintToken = await store.getState().web3Connect.augmint.token;
+    augmintToken.instance.events.AugmintTransfer({}, (error, event) => {
+        // Workaround for bug that web3 beta 36 fires events 2x with MetaMask TODO: check with newer web3 versions if fixed
```

Comment:
Is this duplicate events bug still relevant?

Reply:
Test on localhost ganache (with and without `--blockTime`) and rinkeby too: 
it's triggered multiple times
Although on localhost ganache without Metamask it works fine.
I would leave this monkey fix until we upgrade or replace web3js

Reply:
Ok. In a future round let’s add this workaround to augmint-js, so it doesn’t pollute the client code.